easybuild/easyconfigs/c/cddlib/cddlib-0.94j-GCCcore-8.3.0.eb

```
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'cddlib'
+version = '0.94j'
+
+homepage = 'https://github.com/cddlib/cddlib'
+description = "An efficient implementation of the Double Description Method"
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'cddlib'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['f54bba785b9f6814be5cf6d76861dd61082997c7b48e0244fc3ccea7195892d4']
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('binutils', '2.32'),
+]
+
+dependencies = [('GMP', '6.1.2')]
+
+preconfigopts = "autoreconf -f -i && "
+
+buildopts = "SUBDIRS='lib-src src'"  # build sources but spare the documentation in latex
```

Looks like `make install` triggers the build of docs as well. We have installopts right?
```suggestion
buildopts = "SUBDIRS='lib-src src'"  # build sources but spare the documentation in latex
installopts = buildopts
```

---

@Micket good catch, thanks for looking into it